### DISCUSSION TITLE
Please guide me how to populate multi-select data from server side api

### DISCUSSION THREAD

**Discussion by lytechxdigital** [react-querybuilder/react-querybuilder#757] [Q&A]
Hello sir i need one quick help how to get data from server side as i want to populate all word city data so when i add all data at once my system hang so i want to get all from api ... as i press any word  relected 20 data will get from api and polulate as dropdown .. thanks

**Comment by jakeboone02** [comment]
Honestly this question has many, _many_ different answers and I wouldn't feel comfortable recommending a particular approach without being familiar with your codebase and your team's capabilities and previous experience.
